mungegithub/Makefile

```diff
@@ -109,9 +109,11 @@ push_volume: volume
 ifneq ("", "$(and $(wildcard $(APP)/pv.yaml), $(wildcard $(APP)/pvc.yaml))")
 	# Creating persistent volume and persistent volume claim.
 	kubectl --kubeconfig=$(KUBECONFIG) create -f $(APP)/local.pv.yaml
+	# Sleeping for 1s to allow persistent volume to be created.
+	@sleep 1
 	kubectl --kubeconfig=$(KUBECONFIG) create -f $(APP)/local.pvc.yaml
-	# Sleeping for 2s to allow claim to bind to volume.
-	@sleep 2
+	# Sleeping for 3s to allow claim to bind to volume.
```

Comment:
What is the average during? Rule of thumb is to make this value  2x-3x that value.